### DISCUSSION TITLE
what databases are supported

### DISCUSSION THREAD

**Discussion by zhanghongxia4725** [apache/skywalking#10714] [Q&A]
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/skywalking/issues?q=is%3Aissue) and found no similar feature requirement.

### Description

what kinds of database are supported by skywalking ,if  there is  an unusual databse, how can I make it supported by skywalking to  monitor?

### Use case

what kinds of database are supported by skywalking ,if  there is  an unusual databse, such as Dameng ,how can I make it supported by skywalking to  monitor?

### Are you willing to submit a PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

**Comment by wu-sheng** [comment]
What kind of monitoring do you want to add? Tracing for client access(virtual database), metrics of server(like AWS s3), or logs collecting and analysis.

**Reply by zhanghongxia4725** [reply]
I want monitor sql execute details，I discoveried that if the DB is postgresql, mysql，there is sql message ,but when if the DB is Dameng, there is nothing sql message

**Reply by wu-sheng** [reply]
You don't answer my question. Which features do you refer about database?

**Reply by zhanghongxia4725** [reply]
I  have tried skywalking for application performance monitor, can have a  scale about some slow sql  , but when the DB is Dameng,there is nothing.

**Reply by wu-sheng** [reply]
So, you mean some agents? Such as Java agent?

If so, it means there is no plugin support for your database driver. You need to write it on your own, or ask your vendor to provide that.

**Reply by zhanghongxia4725** [reply]
Yes, java  agent!

**Reply by wu-sheng** [reply]
Reach your vendor. That is a commercial software, not open source. 
We have nothing to do.
